SURVEY.md

```diff
@@ -0,0 +1,145 @@
+# Pandas Devolopers Survey
+
+**Hello folks ,
+
+We need a survey to gather information about the pandas dev. The survey must include information about  general pandas usage, types of development, technologies, and demographics etc.
```

Comment:
Can you clarify who the survey will be distributed to? Is it _all_ pandas contributors?